### PR TITLE
Travel method selection page desktop

### PR DESCRIPTION
**Pull request by Amutha37**
Please review travel method selection page for desktop page 
1. form/TravelMethod.jsx
2. Trello link  : https://trello.com/c/z50v3GrA/297-dev-travel-method-selection-page
3. Importing of transportMode and travelIcon are removed form travelMethodPage. 

*ADDITIONAL INFORMATION 
PR70 should be reviewed first and merge.
PR69 to be reviewed and merged after PR70 .

IMPORTANTION
For the mobile version we still need to work on  wrapping the text on the button.

### CODE REVIEW

**Comment by Amutha37** [comment]
PR 69 depends on PR67/PR68, and that PR67/PR68 should be reviewed/approved/merged before reviewing PR69.

File: pages/form/TravelMethod.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+          <Text fontSize="16px" textAlign={"center"} mb={5}>
+            Please tell us how you travel to work on particular days.
+          </Text>
+          <FormControl
+            isRequired
+            border=".1px solid"
+            width={["305px", "708px"]}
+            height={["400px", "503.37px"]}
+            borderColor={["white", "gray.200"]}
+            // width={"fit-content"}
```
Comment (hqtan) [inline pages/form/TravelMethod.jsx:75]:
```suggestion
```

**Review by hqtan — APPROVED** [review]
happy dance

**Comment by hqtan** [comment]
FYI, 

for `TravelMethodButtons` component, icon and text within button are not centered.
I've added it to our [TODO list](https://trello.com/c/Nuhvxjes) to fix this before survey release.

**Pull request merged**